samples/net/ptp/slave/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(ptp_slave)
+
+file(GLOB APP_SOURCES src/*.c)
+file(GLOB APP_HEADERS src/*.h)
```

Comment:
adding header files to app is not normal.

Either stop doing this, or add a comment in the build script justifying it.